src/scss/admin.scss

```diff
@@ -870,7 +870,7 @@ input.classifai-button {
 }
 
 // Start: Classify Post Modal
-.classify-post-componenet {
+.classify-post-component {
```

Comment:
In doing a quick search to see if the `classify-post-componenet` class was used elsewhere (and thus may need updated) it doesn't appear so. Seems all other places use the proper class name. What this means though is this style has not been applying so we'll need to do some QA here to ensure this style doesn't cause any problems